main.nf

```diff
@@ -840,15 +840,15 @@ process mirdeep2 {
 
     script:
     """
-    perl -ane 's/y/N/ig;print;' $hairpin > hairpin_yn.fa
+    awk '{ gsub(/Y/,"N",\$0); gsub(/B/,"N",\$0); gsub(/K/,"N",\$0); gsub(/M/,"N",\$0); gsub(/R/,"N",\$0); gsub(/S/,"N",\$0); gsub(/W/,"N",\$0); print}' $hairpin > hairpin_ok.fa
```

Comment:
I think you're just adding other nucleotides beyond `Y` here, right? Is there a reason that we can't just do:

```bash
perl -ane 's/[ybkmrsw]/N/ig;print;' $hairpin > hairpin_yn.fa
```

I'd be inclined to use this type of command instead as it's a lot more succinct / easy to read and also case-insensitive..

Reply:
Also I'm not sure I see where you are removing spaces in the headers? Am I missing that somewhere?

Reply:
thanks for having a look

I am fine with the perl command instead of mine but maybe use a different name for the output file as we do not only remove Y nt

I am removing spaces in  headers for the genome file, not the hairpin file, it is at another location in the code

Reply:
> I am removing spaces in headers for the genome file, not the hairpin file, it is at another location in the code

Right, that's what I was wondering - but I can't see that in this pull request, right?

Reply:
I have now added the actual command to remove spaces in genome headers